packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -207,6 +211,43 @@ export class IosSimulatorDevice extends DeviceBase {
     ]);
   }
 
+  private pressingMetaKeys = 0; // 0 - not pressing, 1+ - how many meta keys are pressed
+
+  public sendKey(keyCode: number, direction: "Up" | "Down"): void {
+    // iOS simulator has a buggy behavior when sending cmd+V key combination.
+    // It sometimes triggers paste action but with a very low success rate.
+    // Other times it kicks in before the pasteboard is filled with the content
+    // therefore pasting the previously compied content instead.
+    // As a temporary workaround, we disable sending cmd+V as key combination
+    // entirely to prevent this bugge behavior. Users can still paste content
+    // using the context menu.
+    // This is not an ideal workaround as people may still trigger cmd+v by
+    // pressing V first and then cmd, but it is good enough to filter out
+    // the majority of the noisy behavior since typically you press cmd first.
+    if (keyCode === LEFT_META_HID_CODE || keyCode === RIGHT_META_HID_CODE) {
+      this.pressingMetaKeys += direction === "Down" ? 1 : -1;
```

Comment:
This is not a good approach as it assumes that meatKey "up" event is going to happen, but if it won't it will brak the v key forever. 

How to reproduce a problem: 
1) Click `cmd` with Simulator focused 
2) still holding `cmd` key click on something outside the ide and the release the `cmd` key 
3) the `v` key is now unusable

Reply:
Doesn't the described scenario break all the keys though? In this case the simulator would think that the meta key is pressed the whole time so other keys shouldn't be interactive as well. I don't think fixing this is within the scope of this PR

Reply:
you are correct that this is a larger problem, but in the more general example the keyboard recovers after pressing and releasing `cmd` key, but this solution introduces a new bug that in this scenario disables exclusively the `v` key

Reply:
I updated the code to use bool flags instead of the counter, so now you'll be able to recover from this issue by pressing meta key again